src/main/java/edu/greenblitz/robotName/commands/arm/Elbow/MoveElbowToAngle/MotorMoveElbowToAngle.java

```diff
@@ -0,0 +1,23 @@
+package edu.greenblitz.robotName.commands.arm.Elbow.MoveElbowToAngle;
+
+import edu.greenblitz.robotName.commands.arm.Elbow.ElbowCommand;
+
+public class MotorMoveElbowToAngle extends ElbowCommand {
+
+    private double goalAngle;
```

Comment:
goalAngle?

![image](https://github.com/GreenBlitz/CRRRRESCENDO/assets/80216549/f07e6f01-94e5-4e7c-8bb4-1298594223a8)

Consider changing to `targetAngle`